curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.english.md

```diff
@@ -90,7 +90,33 @@ tests:
 <section id='solution'>
 
 ```js
```

Comment:
```suggestion
```html
```

Reply:
Good catch. Also, it's weird that it works.